package.json

```diff
@@ -229,5 +229,8 @@
     "**/test/fixtures/**",
     "services/web-server/build"
   ],
-  "heroku-run-build-script": true
+  "heroku-run-build-script": true,
+  "postUpdateOptions": [
+    "gomodTidy"
+  ]
```

Comment:
This will need to be within the `renovate` property just above.

Reply:
I'm an idiot.